x/supplier/client/cli/tx_submit_proof.go

```diff
@@ -0,0 +1,55 @@
+package cli
+
```

Comment:
Need to add tests for executing the CLI command. Please add `TODO(@bryanchriswhite)` for it.

See #59 for an example